styles.css

```diff
@@ -190,85 +313,147 @@ body {
     background-color: #fff;
     border-radius: 0 100px 0 0 / 100px;
     padding-top: 8%;
-    padding-right: 4%;
     margin-top: 16%;
   }
 
-  .languages {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 20px;
-    line-height: 24px;
-    color: #000;
-    margin-left: 24px;
-    margin-top: 0;
-    margin-bottom: 0;
-    list-style-type: none;
-    padding-left: 0;
+  .aboutmyself {
+    margin-left: 8vw;
+    margin-right: 8vw;
   }
 
-  .prlanguages {
-    list-style-type: none;
-    padding-left: 0;
+  .contactform {
+    display: flex;
+    flex-direction: column;
+    margin-top: 50px;
+    gap: 50px;
+    background: #6070ff;
+    background-image: url(icons/contact-form-shapes-mobile@2x.png);
+    background-size: 100vw 100vh;
+    border-radius: 100px 0 0 0 / 100px;
   }
 
-  .prlanguages li {
+  .form {
     display: flex;
-    align-items: center;
-    background: #f7f7f9;
+    flex-direction: column;
+  }
```

Comment:
- [ ] It seems like the changes you introduced have created some other issues as the spacing between the form input elements are not visible on your design, see below 👇
  ![image](https://user-images.githubusercontent.com/104968236/237013434-0b06dc10-7049-40ad-a487-4706a11d7459.png)
- [ ] You could add a `gap: 7px;` to the above class and that should fix it. ✔

Reply:
It is really working very well on my computer without the requested changes but I will add the gap: 7px though.